core/src/main/resources/reference.conf

```diff
@@ -325,9 +333,17 @@ cassandra-snapshot-store {
   keyspace = "akka_snapshot"
 
   # Parameter indicating whether the snapshot keyspace should be auto created
+  # This is convenient in development and test environment, but for production
+  # it's recommended to disable this and create the keyspace and tables manually
+  # (with sa cript) once. Not all Cassandra settings are configurable when using
+  # autocreate.
```

Comment:
This seems a bit hand-wavey to me - "Not all Cassandra settings are configurable when using autocreate" is a bit vague (which settings do we suggest changing?), and if that matters I'd generally recommend doing the same for development/test.

Why would we recommend against autocreating for production? It seems quite convenient and hard to mess up.

Reply:
ok, I have reworded. There are many many settings that you may tune for a C* production environment and we have got requests to add them to the autocreate statements, but that will not be possible to maintain.

C* cluster is also not very good at dealing with concurrent creation (from different nodes) of keyspace and tables.